frontend/src/components/TodoList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Todo } from "../models/todo.model";
+
+import TodoListItem from "./TodoListItem";
+
```

Comment:
improve formatting by grouping relative imports together, the space in between is unnecessary